examples/src/test/scala/com/gu/tableversions/examples/MultiPartitionTableLoaderSpec.scala

```diff
@@ -17,6 +17,8 @@ import org.scalatest.{FlatSpec, Matchers}
   */
 class MultiPartitionTableLoaderSpec extends FlatSpec with Matchers with SparkHiveSuite {
 
+  spark.sparkContext.setLogLevel("ERROR")
+
```

Comment:
Might have been mentioned elsewhere, but we could move these to the `TableLoader` class used by all these specs.